backend/src/database/database.providers.ts

```diff
@@ -0,0 +1,12 @@
+import * as mongoose from 'mongoose';
+import { DB_PROVIDER } from '../constants';
+
+export const databaseProviders = [
+    {
+        provide: DB_PROVIDER,
+        useFactory: async () => {
+            (mongoose as any).Promise = global.Promise;
+            return await mongoose.connect('mongodb://localhost:27017/test');
```

Comment:
There still is a switch for mongodb://mongo missing in this line. You might want to take a look on our localhost solution for the frontends Rest client.